openssl/src/crypto/rsa.rs

```diff
@@ -162,23 +162,23 @@ impl RSA {
         }
     }
 
-    pub fn size(&self) -> Result<u32, SslError> {
+    pub fn size(&self) -> Result<isize, SslError> {
```

Comment:
This may make more sense to return `usize` as all uses of this method cast it to `usize` or `c_uint`.